pkg/driver/camera/camera_linux.go

```diff
@@ -25,11 +26,16 @@ type camera struct {
 }
 
 func init() {
-	// TODO: Probably try to get more cameras
-	// Get default camera
-	defaultCam := newCamera("/dev/video0")
-
-	driver.GetManager().Register(defaultCam)
+	searchPath := "/dev/v4l/by-path/"
+	devices, err := ioutil.ReadDir(searchPath)
+	if err != nil {
+		// No v4l device.
+		return
+	}
+	for _, device := range devices {
+		cam := newCamera(searchPath + device.Name())
+		driver.GetManager().Register(cam, device.Name())
```

Comment:
It's impossible to determine the camera label like `front camera` on Linux.
Number in `/dev/video*` and `/dev/v4l/by-id/` depends on the timing of device plug-in.
For these reasons, I put device path (for example `pci-0000\:00\:14.0-usb-0\:9.3\:1.0-video-index0`), which is unique according to the USB port, as a label.

Reply:
I'm about this. On my laptop, I have 2 registered cameras, `/dev/video0` and `/dev/video1`. But, `/dev/video1` is basically not usable, I couldn't even get its information using v4l cli. Do you know why?

Reply:
It is reported on Kernel.org Bugzilla.
https://bugzilla.kernel.org/show_bug.cgi?id=199575
Second one seems for taking metadata of the camera.

Reply:
Ah, I see. Thanks ^^